vault/logical_system.go

```diff
@@ -4497,19 +4497,12 @@ func (b *SystemBackend) pathInternalOpenAPI(ctx context.Context, req *logical.Re
 		return nil, err
 	}
 
-	context := d.Get("context").(string)
-
```

Comment:
If you remove this, you must also remove the declaration of the field at https://github.com/hashicorp/vault/blob/159b60a181bae2d54c666ad09ca607988e4bf5c5/vault/logical_system_paths.go#L955-L958, or ironically, the OpenAPI will wrongly document the OpenAPI-generating operation.

Reply:
Good catch 👍 I'll remove it.

Reply:
On a second thought, I will keep the `context` here since I'm keeping the `CreateOperationIDs` function.